app/src/main/java/io/appium/uiautomator2/model/internal/CustomUiDevice.java

```diff
@@ -166,7 +166,33 @@ public Object findObject(Object selector) throws ClassNotFoundException, Element
             }
             // Prior to API level 21 we can only access the active window
         } else {
-            ret.add(mInstrumentation.getUiAutomation().getRootInActiveWindow());
+            AccessibilityNodeInfo node = mInstrumentation.getUiAutomation().getRootInActiveWindow();
+            if (node != null) {
+                ret.add(node);
+            } else {
+                /*TODO: As we can't proceed to find element with out root node,
```

Comment:
Current implementation is a workaround, need to investigate more for the route cause of the issue and for better understanding mentioned wrote little in detail.